includes/update_handles.batch.inc

```diff
@@ -54,12 +56,12 @@ function islandora_handle_update_handles_batch_operation(array $params, &$contex
         '@pidlist' => $params['pidlist'],
       )));
     }
-    module_load_include('inc', 'islandora_handle', 'includes/handle');
     $sandbox['current'] = 0;
+    $sandbox['pointer'] = ftell($sandbox['file_handle']);
```

Comment:
Don't really have to `ftell()` here... can assume it's at `0` (or will be), with the `fopen()`; however... as [previously indicated](https://github.com/discoverygarden/islandora_handle/pull/27#discussion_r178537925), the `fopen()` should probably be done outside of this `if` bit, since the resources in PHP (as returned by `fopen()`) cannot be serialized... everything in `$context`/`$context['sandbox']` needs to be serializable in order to live across multiple invocations.

Reply:
I mean... perhaps they might do some magic in the drush side, such that it can work (in how it forks off processes/handles its interprocess comms), but if this were to ever be triggered from the GUI somehow, things would likely go kaboom.